GVFS/GVFS.Common/Prefetch/Jobs/ReadFilesJob.cs

```diff
@@ -44,11 +44,24 @@ protected override void DoWork()
                     foreach (string path in this.blobIdToPaths[blobId])
                     {
                         bool succeeded = false;
-                        using (SafeFileHandle handle = NativeFileReader.Open(path))
+
+                        // TODO(Mac): Replace this with a native method as opposed to a slow .NET method

```

Comment:
Nit: this comment is out of place. Either move it into the Mac else branch, or even just drop it. The `IsUnderConstruction` flag already serves as a TODO reminder.